lib/index.js

```diff
@@ -67,12 +67,12 @@ class GraphQLComponent {
         continue;
       }
 
-      const { component, exclude } = imp;
+      const { component, exclude, proxyImportedResolvers = true } = imp;
 
       if (!exclude || !exclude.length) {
         importedDirectives.push(getImportedDirectives(this, component));
         importedTypes.push(...getImportedTypes(this, component));
-        importedResolvers.push(getImportedResolvers(component));
+        importedResolvers.push(getImportedResolvers(component, proxyImportedResolvers));
```

Comment:
do we need to pass this flag everywhere that we call getImportedResolvers from the constructor?

Reply:
The main intention of this flag is to use at the top level for the final schema (i.e. a component that is only made up of imports).

Reply:
ah yes like the one in `hapi-graphql-bootstrap` ... makes sense!